library/src/webapp/js/sakai-table-toolbar/searchFilterPanelMacro.js

```diff
@@ -32,6 +32,8 @@ VM_SFP.doSearch = VM_SFP.doSearch || function(url, buttonElementId, textElementI
     var buttonElement = document.getElementById(buttonElementId);
     VM_SFP.spinButton(buttonElement);
     var searchText = document.getElementById(textElementId).value;
+    //Plus sign means space in a URL and should be replaced.
+    searchText = searchText.replace("+", "%2B");
```

Comment:
what about other special characters? can we just do a urlencode here? and then a urldecode in SiteAction.java?

Reply:
We can do that but it may have side effects, this macro is used in several places.

Reply:
I agree with Sam here, and not just because he's my boss. If people are going to put + signs in search titles they might just put any old rubbish in. Safer to url encode and then decode. To be honest, I thought we'd fixed this years ago.